wgpu-hal/src/gles/adapter.rs

```diff
@@ -564,6 +564,10 @@ impl super::Adapter {
             super::Workarounds::EMULATE_BUFFER_MAP,
             cfg!(target_arch = "wasm32"),
         );
+        workarounds.set(
+            super::Workarounds::INSTANCE_ATTRIBUTE_IMPLY_DRAW_INSTANCED,
+            cfg!(target_arch = "wasm32") || is_angle, // It would be nice to narrow down which browsers are affected (i.e., use ANGLE).
```

Comment:
Firefox appears to be ok on ANGLE as long as `DrawArraysInstanced` is supported at the driver-level (see the conversation in Matrix), but maybe we could be conservative for now and remove this workaround on `wasm32` when it's fixed in Chrome stable. We could still use it for old ANGLE versions though.